scripts/release_scripts/run_macaron.sh

```diff
@@ -585,6 +585,7 @@ else
 fi
 
 docker run \
+    --platform=linux/amd64 \
```

Comment:
Can you please add a comment explaining why this option is added?

Reply:
It seems it's not possible to add comment in the middle of a command in bash. I have to put this flag in a separate variable and put the comment on top of that. Is that ok?

Reply:
You can just add the comment before the `docker run` command. No need to add a variable.